app/controllers/bookings_controller.rb

```diff
@@ -0,0 +1,47 @@
+class BookingsController < ApplicationController
+  def index
+    @bookings = Booking.all
+  end
+
+  def new
+    @booking = Booking.new
+  end
+
+  def create
+    @booking = Booking.new(booking_params)
+    if @booking.save
+      redirect_to booking_path(@booking)
+    else
+      render :new, status: :unprocessable_entity
+    end
+  end
+
+  def show
+    @booking = Booking.find(params[:id])
+  end
+
+  def edit
+    @booking = Booking.find(params[:id])
+  end
+
+  def update
+    @booking = Booking.find(params[:id])
+    if @booking.update(booking_params)
+      redirect_to booking_path(@booking)
+    else
+      render :edit, status: :unprocessable_entity
+    end
+  end
+
+  def destroy
+    @booking = Booking.find(params[:id])
+    @booking.destroy
+    redirect_to bookings_path, notice: 'Booking was successfully destroyed.'
+  end
+
+  private
+
+  def booking_params
+    params.require(:booking).permit(:start, :end, :price, :description, :space_id, :user_id, :confirmed)
```

Comment:
Quick comment here: we don't want to be permitting all these variables when the user creates a new booking. I'd allow start and end only.